SixTest/CMakeLists.txt

```diff
@@ -101,12 +101,17 @@ foreach(TST IN ITEMS ${SIXTRACK_TESTS})
 	file(GLOB TEST_INPUTS LIST_DIRECTORIES FALSE ${CMAKE_SOURCE_DIR}/SixTest/${TST}/*)
 	#message(STATUS "${CMAKE_SOURCE_DIR}/../SixTest/${TST}/")
 
-	foreach(TSFILE IN ITEMS ${TEST_INPUTS})
-		get_filename_component(TFILE ${TSFILE} NAME)
+	if(NOT BOINC)
+		foreach(TSFILE IN ITEMS ${TEST_INPUTS})
+			get_filename_component(TFILE ${TSFILE} NAME)
+
+			#create_symlink old new
+			add_custom_command(TARGET test_copy_${TST} POST_BUILD COMMAND ${CMAKE_COMMAND} -E create_symlink ${TSFILE} ${CMAKE_BINARY_DIR}/SixTest/${TST}/${TFILE})
+		endforeach()
+	else()
+			add_custom_command(TARGET test_copy_${TST} POST_BUILD COMMAND ${CMAKE_COMMAND} -E create_symlink Sixin.zip ${CMAKE_BINARY_DIR}/SixTest/${TST}/Sixin.zip)
```

Comment:
Indentation: This should be on the level of the `foreach(TSFILE IN ITEMS ${TEST_INPUTS})`